src/chooser.js

```diff
@@ -37,7 +37,8 @@ class Chooser extends LitElement {
   static get properties() {
     return {
       fileDisplayName: { type: String },
-      droppedFile: { type: File },
```

Comment:
Assuming these are linter changes, right?

Reply:
Yes — lit doesn't have an implicit way to convert a property specified via an attribute (as a string) into a `File`. The only place this component is used (in `appmain.ts`) passes in a file is via the dot property setter (which has a name, I'm just forgetting it).